app/form/v1_0/fields/drug-use/drug-use-history.ts

```diff
@@ -31,6 +31,23 @@ const drugsReasonsForUse: FormWizard.Field = {
     { text: 'Other', value: 'OTHER', kind: 'option' },
   ],
   labelClasses: utils.getMediumLabelClassFor(FieldType.CheckBox),
+  transform(state): FormWizard.Field {
+    let fieldText = this.text
+
+    if (
+      Object.entries(state.answers)
+        .filter(([key]) => key.startsWith('drug_last_used_'))
+        .map(([_, answer]) => answer)
+        .every(answer => answer === 'MORE_THAN_SIX')
+    ) {
+      fieldText = fieldText.replace('does', 'did')
+    }
+
+    return {
+      ...this,
+      text: fieldText,
+    }
```

Comment:
We've used this check and pattern in two other places (drug-details.ts and drug-use-history.ts)
Might be better for consistency:

```suggestion
    const usedInTheLastSixMonths = addDrugs.drugLastUsedFields.some(field => state.answers[field.code] === 'LAST_SIX')

    return {
      ...this,
      text: usedInTheLastSixMonths ? this.text : this.text.replace('does', 'did'),
    }
```